core/modules/comment/tests/comment_views_handler_argument_user_uid.test

```diff
@@ -33,15 +34,36 @@ class CommentViewsHandlerArgumentUserUidTest extends ViewsSqlTest {
   }
 
   protected function setUp(array $modules = array()) {
+    $modules[] = 'comment';
     parent::setUp($modules);
 
-    // Add two users, create a node with the user1 as author and another node with user2 as author.
+    // Create a post content type with comments enabled and grant permissions.
+    $this->backdropCreateContentType(array(
+      'type' => 'post',
+      'name' => 'Post',
+      'settings' => array(
+        'comment_default' => COMMENT_NODE_OPEN,
+      ),
+      'is_new' => TRUE,
+    ));
+    user_role_grant_permissions(BACKDROP_AUTHENTICATED_ROLE, array(
+      'access content',
+      'access comments',
+      'post comments',
+      'skip comment approval',
+    ));
+
+    // Add two users, create a node with the user1 as author and another node
```

Comment:
It should be * with user1 as author*, in the same way it is * with user2 as author*.